lib/chunks.sql

```diff
@@ -52,7 +52,9 @@ WITH RECURSIVE template_questions AS (
     FROM
         questions AS q
         JOIN questions AS tq ON (tq.qid = q.template_directory AND tq.course_id = q.course_id)
-    WHERE q.id = $question_id
+    WHERE
+        q.id = $question_id
+        AND q.deleted_at IS NULL
```

Comment:
Wait, damn, I think *this* is it 😭 
```suggestion
        AND tq.deleted_at IS NULL
```